packages/clerk-js/src/ui/lazyModules/providers.tsx

```diff
@@ -128,3 +129,27 @@ export const LazyImpersonationFabProvider = (
     </Suspense>
   );
 };
+
+type LazyOneTapRendererProps = React.PropsWithChildren<
+  {
+    componentProps: any;
+    startPath: string;
+  } & Omit<AppearanceProviderProps, 'appearanceKey'>
+>;
+
+export const LazyOneTapRenderer = (props: LazyOneTapRendererProps) => {
```

Comment:
Why did we introduce `LazyOneTapRenderer` and didn't use `LazyComponentRenderer` (just asking out of curiosity)

Reply:
They are using a different portal, previously the Portal had some conditionals just for managing the one tap. in this PR i've decoupled that by introducing a new component